src/core/search/parser.y

```diff
@@ -55,6 +55,7 @@ using namespace std;
   LCURLBR  "{"
   RCURLBR  "}"
   OR_OP    "|"
+  KNN    "KNN"
```

Comment:
Looks like the alignment here isn't like the others
(that's the only thing I can say about this weird `.y` file :)